shared/common-adapters/text.css

```diff
@@ -1,600 +1,343 @@
 /* Used by text on desktop to speed up perf */
 
-.text_HeaderBig {
```

Comment:
instead of having a ton of dupe stuff, lets just collapse it so its clear which types go with which attributes